core/CHANGELOG.md

```diff
@@ -243,7 +243,7 @@
   superflucious. Both are removed in favor of a derivation through `THandler`
   (see [PR 2183]).
 
-- Require `ConnectionHandler::{InEvent,OutEvent,Error}` to implement `Debug`
+- Require `ConnectionHandler::{InEvent,ToBehaviour ,Error}` to implement `Debug`
```

Comment:
Let's not change changelogs please.

Reply:
Yes of course, I thought I had cleaned everything, this one was missing